sno/cli.py

```diff
@@ -81,7 +81,24 @@ def print_version(ctx):
     ctx.exit()
 
 
-@click.group()
+class PdbGroup(click.Group):
+    def invoke(self, ctx):
```

Comment:
I thought using [`sys.excepthook()`](https://docs.python.org/3/library/sys.html#sys.excepthook) is preferred over a try/catch?

https://stackoverflow.com/a/1237407/2662

Reply:
TIL. I guess using that allows us to catch exceptions that happen after our command exits, i.e. in whatever click does just as the process is about to exit.

Seems odd but okay.

Reply:
I actually like the try/except version better, because it also handles exceptions that will be caught by click (e.g. usage errors). So it comes in useful when you're trying to debug "Error: No commit found at abc132" as well.